frameworks/OCaml/opium/haproxy.cfg

```diff
@@ -0,0 +1,59 @@
+global
+    chroot      /var/lib/haproxy
+    pidfile     /var/run/haproxy.pid
+    maxconn     32768
+    user        haproxy
+    group       haproxy
+    daemon
+
+defaults
+    mode                    tcp
+    log                     global
+    option                  dontlognull
+    option http-server-close
+    option forwardfor       except 127.0.0.0/8
+    option                  redispatch
+    retries                 3
+    timeout http-request    10s
+    timeout queue           1m
+    timeout connect         10s
+    timeout client          1m
+    timeout server          1m
+    timeout http-keep-alive 10s
+    timeout check           10s
+    maxconn                 3000
+
+frontend main
+    bind *:8080
+    default_backend             tfb
+
+backend tfb
+    balance     roundrobin
```

Comment:
I'm going to prepare a PR for webmachine that changes this to `static-rr` in the next few days, you might want to do the same.
https://cbonte.github.io/haproxy-dconv/2.3/configuration.html#4.2-balance

Reply:
did you find it to have better performance?
or do you plan to create a separate configuration to test it out?